apps/common-app/src/App.tsx

```diff
@@ -200,7 +202,7 @@ export default function App() {
     };
 
     if (!isReady) {
-      restoreState();
+      restoreState().catch(noop);
```

Comment:
Not a big deal, but I was seeing this eslint error:

![Screenshot 2024-07-03 at 17 10 31](https://github.com/software-mansion/react-native-reanimated/assets/52978053/eda2864e-be02-4cd0-9cf3-f42ceff46393)